corehq/apps/linked_domain/applications.py

```diff
@@ -92,7 +92,10 @@ def get_linked_apps_for_domain(domain):
     return linked_apps
 
 
-def create_linked_app(master_domain, master_id, target_domain, target_name, remote_details=None):
+def link_app_via_app_manager(master_domain, master_id, target_domain, target_name, remote_details=None):
```

Comment:
How about calling it `legacy_link_app` or `deprecated_link_app` so people don't need to see the comment? If it's going to be a weird name, might as well be crystal clear.

Reply:
(resolved in later commit)